### ISSUE TITLE
sync prints a "{}" error when the response from the sync is a 404 (3 pts)

### ISSUE THREAD

**Issue by daneryl** [huridocs/uwazi#3768]
https://github.com/huridocs/uwazi/blob/c1b495920df123e7d4e9e249b9891201185cccf9/app/api/sync/syncWorker.js#L74

probably prettify error is the responsable, this function should have a default when the received error is not any of the things that expects.

to replicate: configure a sync to an nonexistent url.

**Comment by LaszloKecskes** [comment]
On further investigation, the cause of the issue is different. Completely non-existing url-s trigger a properly formed 'FetchError' in typescript's fetch, that is used in the [JSONRequest module](https://github.com/huridocs/uwazi/blob/95435b127bedda61ce1ba5113aaad53bf4a53d22/app/shared/JSONRequest.js#L83). That is printed without a problem.

Existing url's that do not have the UWAZI sync service up get to [this line](https://github.com/huridocs/uwazi/blob/95435b127bedda61ce1ba5113aaad53bf4a53d22/app/shared/JSONRequest.js#L108), where we throw the object that later is mishandled by prettifyError.

In short, to replicate the issue, configure a sync to an _existing_ url that does not have Uwazi, i.e. `google.com`.
